mivs/templates/mivs_applications/index.html

```diff
@@ -130,7 +131,7 @@
         {% elif needs_screenshots %}
           Please upload at least two screenshots before submitting the video of your game.
         {% elif not game.video_submittable %}
-          Please include a link to a video using the Edit link above, no longer than 2 minutes, showing
+          Please include a link to a video for your game ({{ game_edit_link }}), no longer than 2 minutes, showing
```

Comment:
This might look a little weird, because `game_edit_link` already has brackets around it: `([Edit])`. Maybe drop the parens, so it renders like this: `[Edit]`?